ivy/functional/frontends/jax/nn/non_linear_activations.py

```diff
@@ -99,18 +89,29 @@ def _reduction_dims(a, axis):
         return canon_axis, canon_axis
 
 
-def _mean(x, axis=None, keepdims=False, where=None):
-    # Mean with support for where
-    if where is None:
-        return ivy.mean(x, axis=axis, keepdims=keepdims)
+# Helpers #
+# ------- #
```

Comment:
I think those should be removed, what do you think @AnnaTz

Reply:
Maybe it would be best to have each file separated in 2 sorted parts; move all helpers at the top of the file alphabetically, and then all the actual functions alphabetically.

Reply:
Ideally the frontends should not use helpers functions (as we are targeting to shorten all functions and move the computations to the main ivy api). I don't think there are many frontend files that still have helpers.